.github/workflows/testsuite.yaml

```diff
@@ -220,3 +220,53 @@ jobs:
           file: ./coverage.xml #optional
           env_vars: OS,PYTHON
           fail_ci_if_error: true
+
+  download_test:
```

Comment:
I don't quite understand what this job is adding that is not covered by other jobs?

Reply:
Ah, ok. I had missed the comment above that test that explained how it was prevented from being run on all the other jobs.